### PR TITLE
[5.0] TinyMCE plugins

### PR DESCRIPTION
**Pull request by brianteeman**
Pull Request for Issue # .

### Summary of Changes
When displaying the help page inside tinymce there is a list of the plugins being used and links to more information
This PR updates the three external plugins with a correct name and a link to joomla.org. The link itself can always be updated to a specific page in the future.

### Testing Instructions
Enable the help btton in the tinymce plugin

image

In an article select the help button

### Actual result BEFORE applying this Pull Request
image

### Expected result AFTER applying this Pull Request
image

### Link to documentations
Please select:
- [ ] Documentation link for docs.joomla.org: <link>
- [x] No documentation changes for docs.joomla.org needed

- [ ] Pull Request link for manual.joomla.org: <link>
- [x] No documentation changes for manual.joomla.org needed

### CODE REVIEW

**Comment by wilsonge** [comment]
Should these not be translatable strings as they're being rendered?

**Comment by brianteeman** [comment]
> Should these not be translatable strings as they're being rendered?

i dont think any of the strings on that tab are translated

**Comment by Fedik** [comment]
I have tested this item :white_check_mark: successfully on 39561f86cf2412614493a15e6bb5f8aaea089054<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/41592">issues.joomla.org/tracker/joomla-cms/41592</a>.</sub>

File: build/media_source/plg_editors_tinymce/js/plugins/dragdrop/plugin.es6.js [hunk 1]
```diff
@@ -168,4 +168,10 @@ window.tinymce.PluginManager.add('jdragndrop', (editor) => {
 
     editor.contentAreaContainer.style.borderWidth = '0';
   });
+  return {
+    getMetadata: () => ({
+      name: 'Drag and Drop (Joomla)',
+      url: 'https://www.joomla.org/'
+    })
```
Comment (richard67) [inline build/media_source/plg_editors_tinymce/js/plugins/dragdrop/plugin.es6.js]:
```suggestion
      url: 'https://www.joomla.org/',
    }),
```
Fix javascript-cs failing in drone.

**Comment by ceford** [comment]
I must be doing something wrong. With 5.0.0-beta-2 I apply the patch and npm ci - I get a different set of Help links but not like in the illustration:

screen shot 2023-09-12 at 04 37 56<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/41592">issues.joomla.org/tracker/joomla-cms/41592</a>.</sub>

**Comment by ceford** [comment]
I have tested this item :white_check_mark: successfully on 159deff113946a4c1774ba63ea7afb1a16007a06<br /><br />For my previous test I neglected to bring my clone up to data. I now see the links as in the illustration.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/41592">issues.joomla.org/tracker/joomla-cms/41592</a>.</sub>

**Comment by ceford** [comment]
I have not tested this item.<br /><br />For my previous test I neglected to bring my clone up to data. I now see the links as in the illustration.

Actually - two new links point to docs.joomla.org - surely that should be fixed before approval.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/41592">issues.joomla.org/tracker/joomla-cms/41592</a>.</sub>

**Pull request merged**